blivet/devicetree.py

```diff
@@ -1552,7 +1552,14 @@ def addLV(lv):
                         lv_device.cache = LVMCache(lv_dev, cache_size, exists=True)
                 self._addDevice(lv_device)
                 if flags.installer_mode:
-                    lv_device.setup()
+                    try:
+                        lv_device.setup()
+                    except LVMError:
+                        # The only scenario in which this is expected is an incomplete VG
+                        # and/or multiple VGs with the same name.
+                        log.warning("failed to activate lv %s -- ignoring it", lv_device.name)
+                        self._removeDevice(lv_device)
```

Comment:
This is the only way I could see that doesn't involve adding new handling for devices that exist but for some reason cannot be activated. It's probably necessary to acknowledge the device exists, though -- with this patch we would lose the ability to remove the LV later.